config/samples/csi.ibm.com_v1_hostdefinition_cr.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: csi.ibm.com/v1
+kind: HostDefinition
+metadata:
+  name: ibm-block-csi
+  labels:
+    app.kubernetes.io/name: ibm-block-csi
+    app.kubernetes.io/instance: ibm-block-csi
+    app.kubernetes.io/managed-by: ibm-block-csi-operator
+    release: v1.10.0
+spec:
+  hostDefinition:
+    managementAddress: demo-management-address
+    hostNameInStorage: demo-hostNameInStorage
+    #connectivityPorts:
+    #  iscsi:
+    #    - iqn1
+    #    - iqn2
+    #  fc:
+    #    - wwn1
+    #    - wn2
+    #  nvme:
+    #    - nqn1
+    #    - nqn2
+    #secretName: demo-secret
+    #secretNamespace: default
+    #connectvityType:
```

Comment:
please use a spell checker

Reply:
`connectvityType` is no longer in hostDefinition crd